src/runtime/flow_node_handler/activity_handler/activity_handler.ts

```diff
@@ -442,6 +442,13 @@ export abstract class ActivityHandler<TFlowNode extends Model.Base.FlowNode> ext
     processModelFacade: IProcessModelFacade,
   ): Array<IFlowNodeModelInstanceAssociation> {
 
+    const getBoundaryEventPreceedingFlowNodeInstance = (flowNodeInstance: FlowNodeInstance): Model.Events.BoundaryEvent => {
```

Comment:
Auch wenn du das nur verschoben hast... hier ist ein Typo 😄
(Da wo die Funktion benutzt wird müsste das auch entsprechend angepasst werden)

```suggestion
    const getBoundaryEventPrecedingFlowNodeInstance = (flowNodeInstance: FlowNodeInstance): Model.Events.BoundaryEvent => {
```